cmd/ocsp-updater/main.go

```diff
@@ -33,16 +33,30 @@ type Config struct {
 		// E1 -> 1, R3 -> 3, etc.
 		Issuers map[string]int
 
-		OldOCSPWindow    cmd.ConfigDuration
+		// OldOCSPWindow controls how frequently ocsp-updater signs a batch
+		// of responses.
+		OldOCSPWindow cmd.ConfigDuration
+		// OldOCSPBatchSize controls the maximum number of responses
+		// ocsp-updater will sign every OldOCSPWindow.
 		OldOCSPBatchSize int
 
-		OCSPMinTimeToExpiry          cmd.ConfigDuration
+		// The worst-case freshness of a response during normal operations.
+		// This is related to to OCSPMinTimeToExpiry in ocsp-updater's config,
```

Comment:
```suggestion
		// This is related to to ExpectedFreshness in ocsp-responder's config,
```

Reply:
Changed `ocsp-updater` to `ocsp-responder`.